caffe2_builders/v140/auto.bat

```diff
@@ -0,0 +1,102 @@
+rem Remove to original folder after script is finished
+set ORIGINAL_DIR=%cd%
+
+rem Build folders are 
+rem %CAFFE2_ROOT%\build\Debug for Debug and
+rem %CAFFE2_ROOT%\build\Release for Release
+set CAFFE2_ROOT=%~dp0%\pytorch
+
+rem Cloning repository if it doesn't exist
+if not exist pytorch (
+    set PYTORCH_BRANCH=windows-shared-build
+    set PYTORCH_REPO=ArutyunovG
+    call  %~dp0%..\..\internal\clone.bat
+    cd %ORIGINAL_DIR%
+)
+
+rem Should build folder be deleted and build start from scratch?
+if NOT DEFINED REBUILD (
+    set REBUILD=0
+)
+
+rem Debug build enabled by default
+if NOT DEFINED BUILD_DEBUG (
```

Comment:
So if both `BUILD_DEBUG` and `BUILD_RELEASE` are not specified, `DEBUG` will be used. I think it should default to Release instead.

Reply:
It must build both, Debug and Release.

Reply:
First it builds debug, then release.